docs/developing.md

```diff
@@ -10,6 +10,10 @@ $ make dev
 
 You can view the pattern library at http://localhost:4001/
 
+The pattern library is also publicly available in https://patterns.hypothes.is/, where it gets automatically updated every time a new version is released.
+
+It can also be manually deployed using the [deploy workflow](https://github.com/hypothesis/frontend-shared/actions/workflows/deploy.yml)
+
```

Comment:
Just a bit of suggested tightening here:

```suggestion
The pattern library is hosted at https://patterns.hypothes.is/. It gets deployed automatically when a new package version is published.

It can be manually deployed using the [deploy workflow](https://github.com/hypothesis/frontend-shared/actions/workflows/deploy.yml)

```